climate/toon.py

```diff
@@ -40,8 +35,10 @@ def __init__(self, hass):
         self._state = None
         self._temperature = None
         self._setpoint = None
-        self._operation_list = [STATE_HEAT, STATE_HOME, STATE_NOT_HOME,
-                                STATE_COOL, STATE_UNKNOWN]
+        self._operation_list = [STATE_PERFORMANCE,
+                                STATE_HEAT,
+                                STATE_ECO,
+                                STATE_COOL]
```

Comment:
I'm not very happy with this change, but it needed to be done to be able to merge. As the hass team pointed out, all new climate components need to use these states and they enable better control of toon by scripts/automations. Half way though the pull request some [new states](https://github.com/home-assistant/home-assistant/commit/312de6b3a369bdc58dda94b2a78167ef38821cee#diff-b9607e70a7ce29fdc462c9457fd38bbfR49) were added to the core, so i've used them to get as close as possible to the actual toon states. 

I can imagine you'd want to keep the normal native toon states in this repo though.

Reply:
No, I don't mind at all. This is absolutely fine and it makes sense to use common states across common devices. The previous time we tried to get his in there were not enough states that made sense to describe the usability so basically toon was getting crippled.